web/public/mgmt/config.yml

```diff
@@ -135,95 +135,104 @@ collections:
+            display_fields: ["{{id}}"]
+            required: false
+          - { label: "Required", name: "required", widget: "boolean", required: false }
+          - { label: "Issuing Agency", name: "issuingAgency", widget: "string", required: false }
+          - { label: "Form Name", name: "formName", widget: "string", required: false }
+      - label: "Annual License Task Metadata"
+        name: "applyForAnnualLicense-task"
+        file: "content/src/roadmaps/tasks/annual-license-cannabis.md"
         identifier_field: "{{fields.slug}}"
         slug: "{{fields.slug}}"
         editor:
           preview: true
         fields:
           - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Social Equity Requirements"
-        name: "social-equity-requirements"
-        file: "content/src/display-content/cannabis-license/social-equity-requirements.md"
-        identifier_field: "{{fields.slug}}"
-        slug: "{{fields.slug}}"
+          - { label: "Url Slug", name: "urlSlug", widget: "string" }
+          - { label: "Filename", name: "filename", widget: "slug", required: false }
+          - { label: "Name", name: "name", widget: "string" }
+          - { label: "Id", name: "id", widget: "string" }
+          - { label: "Call To Action Link", name: "callToActionLink", widget: "string", required: false }
+          - { label: "Call To Action Text", name: "callToActionText", widget: "string", required: false }
+          - label: Industry
+            name: industryId
+            widget: "relation"
+            collection: "roadmaps"
+            search_fields: ["{{id}}"]
+            value_field: "{{id}}"
+            display_fields: ["{{id}}"]
+            required: false
+          - { label: "Required", name: "required", widget: "boolean", required: false }
+          - { label: "Issuing Agency", name: "issuingAgency", widget: "string", required: false }
+          - { label: "Form Name", name: "formName", widget: "string", required: false }
+
+      - label: "Tab 1 (Business Size and Priority Status)"
+        name: "cannabisLicense-1"
+        file: "content/src/fieldConfig/cannabis-license-tab1.json"
         editor:
           preview: true
         fields:
-          - { label: "Content", name: "body", widget: "markdown" }
-      - label: "Config"
-        name: "config"
-        file: "content/src/fieldConfig/config.json"
-        editor:
-          preview: false
-        fields:
-          - label: Cannabis Apply For License
+          - label: Business Size and Priority Status
             name: cannabisApplyForLicense
             collapsed: false
             widget: object
             summary: "{{cannabisApplyForLicense}}"
             fields:
+              - {
+                  label: Priority Status None Selected Text,
+                  name: priorityStatusNoneSelectedText,
+                  widget: string,
+                  hint: This field supports Markdown,
```

Comment:
i think we should make this one a markdown widget, because it has a link in it, so it'll be easier for jess and ce to edit if they can use the rich text editor

Reply:
actually I think this comment holds for pretty much every field here in the first tab, because they all have contextual info or links, and it's much easier for them to edit those with the rich text markdown editor